frontend/src/components/task.jsx

```diff
@@ -22,16 +22,13 @@ export default function Task(props) {
     };
 
     const deleteTask = async (taskId) => {
-        const res = await fetch(`http://localhost:3001/tasks/${taskId}`, {
+        await fetch(`http://localhost:3001/tasks/${taskId}`, {
             method: "DELETE"
         });
-        const json = await res.json();
-        if (json.acknowledged) {
```

Comment:
There's no `acknowledged` field in the response JSON, so we just remove this check here. You already had the logic in here to filter the removed Task from the `tasks` state variable, so we'll left it there to make this deletion work in the UI :)